### PR TITLE
Okoro

### PR DESCRIPTION
**Pull request by Overbugger**
Set up app, added a frontend and built a registration endpoint

### CODE REVIEW

File: Backend/controllers/auth/auth.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      req.body.password,
+      foundUser.rows[0].password
+    );
+    if (hashedPassword === false) {
+      return res.json({ message: "Invalid" });
+    }
+    const token = jwt.sign({ username }, secret, {
+      expiresIn: 60 * 60,
+    });
+    console.log(res.status)
```
Comment (salimkarbm) [inline Backend/controllers/auth/auth.js]: remove console log

**Review by salimkarbm — COMMENTED** [review]
remove all the commented code

File: Backend/controllers/auth/auth.js (hunk 1 above)
Comment (Overbugger) [reply Backend/controllers/auth/auth.js]: yes sir

**Comment by Overbugger** [comment]
@salimkarbm

**Pull request merged**